README.md

```diff
@@ -1,143 +1,94 @@
-# persistenceCore
+![Persistence Core](https://www.asiacryptotoday.com/wp-content/uploads/2020/09/persistence-protocol.jpg)
```

Comment:
Please update the README in a separate PR. Changing CI pipeline for Docker is a functional concern, the README text is non-functional concern, we should separate such things in different PRs.

Reply:
Oh, and don't reference JPEGs from 3rd party websites please in any file of the official codebase